lib/include/openamp/remoteproc.h

```diff
@@ -395,6 +395,13 @@ struct remoteproc {
  *        memory may not be off.
  * @shutdown: shutdown the remoteproc and release its resources.
  * @notify: notify the remote
+ * @get_mem: get remoteproc memory I/O region by either name, virtual address,
+ *           physical address or device address.
+ *           Note:
+ *           1.remoteproc_get_io_with_xxx try the internal list added by
+ *             remoteproc_add_mem first and then get_mem.
```

Comment:
Put this comment in  remoteproc_get_mem as this is related to remoteproc_get_mem  behavior not the op

Reply:
Done.